src/find/azext_find/custom.py

```diff
@@ -0,0 +1,55 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import random
+import json
+import requests
+
+from knack.log import get_logger
+logger = get_logger(__name__)
+
+wait_messages = ['Ok, let me find an answer to that question for you.',
+                 'I\'m working on finding the right answer for you.', 'Let me see if I answer that for you.']
+
+
+def processquery(question):
+    print(random.choice(wait_messages))
+    logger.warn('Please wait...\n')
+    process_answer(question)
+
+
+def process_answer(query):
+    answer_list = call_aladdin_service(query)
+    num_results_to_show = 1
+    if answer_list:
+        if answer_list[0]['source'] == 'bing':
+            print("Here are some information I was able to gather for you: ")
+            num_results_to_show = 3
+        for i in range(num_results_to_show):
+            print(answer_list[i]['title'])
+            print(answer_list[i]['snippet'])
+            print("For more information please see:", answer_list[i]['link'])
+            if i + 1 < num_results_to_show:
+                print("=========================================")
+
+
+def call_aladdin_service(query):
+    service_input = {
+        'paragraphText': "",
+        'currentPageUrl': "",
+        'query': "ALADDIN-CLI:" + query,
+        'context': ""
+    }
+
+    api_url = 'https://aladdinservice-staging.azurewebsites.net/api/aladdin/generateCards'
+    headers = {'Content-Type': 'application/json'}
+
+    response = requests.post(api_url, headers=headers, json=service_input)
+
+    if response.status_code == 200:
+        answers = json.loads(response.content)
+        return answers
+
+    return response.status_code
```

Comment:
No, returning this here doesn't do anything, the full response needs to be returned by `processquery` since that is what the command calls. 
Since it doesn't make sense to have many different return types for different scenarios, move the status_code checking logic outside of this function and let this function just make the rest call as its name suggests.